nodejs/awsx/apigateway/api.ts

```diff
@@ -370,6 +370,8 @@ export class API extends pulumi.ComponentResource {
 
         // Create the API Gateway Rest API, using a swagger spec.
         this.restAPI = new aws.apigateway.RestApi(name, {
+            name: swaggerString.apply(s => { const spec = JSON.parse(s); return spec.info.title; }),
```

Comment:
We should handle `.info` or `.info.title` being undefined.

May be easiest to just fill it in if needed.

Reply:
Done.